functests/utils/pods/pods.go

```diff
@@ -223,3 +224,27 @@ func GetPerformanceOperatorPod() (*corev1.Pod, error) {
 
 	return &pods.Items[0], nil
 }
+
+//GetCpusNumber returns the cpus number that are used by mco pod
+func GetCpusNumber(pod *corev1.Pod) (int, error) {
+	cmdGetCPU := []string{"bash", "-c", fmt.Sprintf("cat /sys/fs/cgroup/cpuset/cpuset.cpus")}
```

Comment:
I'm not sure this path is correct. It seems to lack the cgroup name at very least.

Reply:
Can you illustrate more? 
I see these groups under cgroup:
[root@oslat-sdzlq cgroup]# ls 
blkio  cpu,cpuacct  cpuset   freezer  memory   net_cls,net_prio  perf_event  rdma
cpu    cpuacct	    devices  hugetlb  net_cls  net_prio		 pids	     systemd